sonar-project.properties

```diff
@@ -10,12 +10,12 @@ sonar.projectVersion=v0.23.0
 
 sonar.sources=.
 
-sonar.exclusions=**/*_test.go,**/vendor/**,**/testdata/*
+sonar.exclusions=**/*_test.go,**/vendor/**,**/testdata/**
 
 sonar.tests=.
 sonar.test.inclusions=**/*_test.go
 sonar.test.exclusions=**/vendor/**
 
-sonar.go.coverage.reportPaths=coverage.out
-sonar.go.golangci-lint.reportPaths=golangci-lint.report.xml
-sonar.go.tests.reportPaths=TEST-unit.xml
+sonar.go.coverage.reportPaths=**/coverage.out
+sonar.go.golangci-lint.reportPaths=**/golangci-lint.report.xml
+sonar.go.tests.reportPaths=**/TEST-unit.xml
```

Comment:
For modules, we are going to need updating this: https://github.com/testcontainers/testcontainers-go/blob/main/commons-test.mk#L19

```diff
- 		--junitfile TEST-$*.xml \
+ 		--junitfile TEST-unit.xml \
```

Reply:
Is the file not always named TEST-unit.xml ?

Reply:
I think not for modules :(